src/main/java/org/openstreetmap/atlas/checks/validation/intersections/LineCrossingWaterBodyCheck.java

```diff
@@ -146,23 +221,43 @@ private static boolean isBoundary(final AtlasEntity crossingLine)
     public LineCrossingWaterBodyCheck(final Configuration configuration)
     {
         super(configuration);
+        this.filterOffendingLineItems = this.configurationValue(configuration,
```

Comment:
Rather than having this as a boolean switch, perhaps it would be more useful to have the configurable be a taggable filter. That way the user could easily select whatever lines they liked to be flagged.

Reply:
Agreed. Done